src/Germany/Mews.Fiscalizations.Germany.Tests/TestFixture.cs

```diff
@@ -1,4 +1,5 @@
-﻿using Mews.Fiscalizations.Germany.Model;
+﻿using Mews.Fiscalizations.Germany.V1;
+using Mews.Fiscalizations.Germany.V1.Model;
 using System;
 
 namespace Mews.Fiscalizations.Germany.Tests
```

Comment:
We could also copy paste the tests, so that tests have a v1 too.

Reply:
I would keep tests for the latest version only.

Having tests for all versions means we have to add more pipelines and maintain all of them.